samples/Exporters/Console/Program.cs

```diff
@@ -111,6 +112,13 @@ internal class ConsoleOptions
         public bool DisplayAsJson { get; set; }
     }
 
+    [Verb("otelshim", HelpText = "Specify the options required to test OpenTelemetry Shim with console exporter")]
+    internal class OpenTelemetryShimOptions
+    {
+        [Option('p', "displayasjson", HelpText = "Specify if the output should be displayed as json or not (default: false)", Default = false)]
```

Comment:
Not blocking this PR, eventually we might want to align with ILogger console logger https://github.com/dotnet/runtime/issues/34742 (e.g. JSON options, formatter, etc.).

Reply:
+1 on this. let me know if you have any questions on this :)